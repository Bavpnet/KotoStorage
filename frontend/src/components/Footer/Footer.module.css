.footer {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  margin-top: 64px;
}

.footerInfo {
  display: flex;
  gap: 115px;
  justify-content: center;
  align-items: center;
}

.line {
  height: 3px;
  width: 100%;
  background-color: #5e564b;
}

.logo {
  display: flex;
  flex-direction: column;
  align-items: center;
}

.img {
  width: min(300px, 20vw);
  height: min(300px, 20vw);
}

.heading3 {
  font-family: 'Qwigley', cursive;
  font-size: 4rem;
  font-weight: 400;
  white-space: nowrap;
}
