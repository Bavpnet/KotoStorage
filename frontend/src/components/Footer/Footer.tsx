import { SocialInfo } from '../SocialInfo'
import Logo from '../../assets/logo.png'
import styles from './Footer.module.css'

export const Footer = () => {
  return (
    <footer className={styles.footer}>
      <span className={styles.line} />
      <div className={styles.footerInfo}>
        <div className={styles.logo}>
          <img src={Logo} alt="logo" className={styles.img} />
          <h3 className={styles.heading3}>"Murr means love"</h3>
        </div>
        <SocialInfo isFooterInfo={true} />
      </div>
    </footer>
  )
}
