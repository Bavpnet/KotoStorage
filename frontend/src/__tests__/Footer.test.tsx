import renderer from 'react-test-renderer'
import { Footer } from '../components/Footer'

describe('Footer', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Footer />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
