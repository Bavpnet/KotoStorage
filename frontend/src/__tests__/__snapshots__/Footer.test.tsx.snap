// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  className="footer"
>
  <span
    className="line"
  />
  <div
    className="footerInfo"
  >
    <div
      className="logo"
    >
      <img
        alt="logo"
        className="img"
        src="logo.png"
      />
      <h3
        className="heading3"
      >
        "Murr means love"
      </h3>
    </div>
    <section
      className="inFooter"
    >
      <h2
        className="infoTitle"
      >
        Follow us in the media
      </h2>
      <p
        className="infoDesc"
      >
        Hear about exclusive news and changes in the cats world
      </p>
      <ul
        className="social"
      >
        <li>
          Instagram
        </li>
        <li>
          Telegram
        </li>
        <li>
          Github
        </li>
      </ul>
    </section>
  </div>
</footer>
`;
